packages/app/app/stacks/stacks-orders.tsx

```diff
@@ -27,8 +28,9 @@ export const StackOrders = ({ chainId, address }: StackOrdersProps) => {
   );
   const activeOrders = filterActiveOrders(currentStackOrders);
   const completedOrders = filterCompletedOrders(currentStackOrders);
+  const cancelledOrders = filterCancelledOrders(currentStackOrders);
```

Comment:
This is just a suggestion

```suggestion
      const ordersByType = [
                { orders: filterActiveOrders(currentStackOrders), emptyText: "No active stacks" },
                { orders: filterCompletedOrders(currentStackOrders), emptyText: "No complete stacks" },
                { orders: filterCancelledOrders(currentStackOrders, emptyText: "No cancelled stacks" },
              ]
```

Reply:
good suggestion, updated